source/features/clean-issue-filters.tsx

```diff
@@ -40,18 +30,30 @@ function removeParent(element?: Element): void {
 	}
 }
 
-async function init(): Promise<void> {
-	const {projects, milestones} = await getCounts();
+async function hideMilestones(): Promise<void> {
+	const hasNoMilestones = select('[data-selected-links^="repo_milestones"] .Counter')!.textContent!.trim() === '0';
 
-	if (projects === 0) {
-		elementReady('[data-hotkey="p"]').then(removeParent);
+	if (hasNoMilestones) {
+		elementReady('[data-hotkey="m"]').then(removeParent);
 	}
+}
 
-	if (milestones === 0) {
-		elementReady('[data-hotkey="m"]').then(removeParent);
+async function hideProjects(): Promise<void> {
+	const projectsTab = select([
+		'[data-hotkey="g b"]', // In organizations and repos
+		'.user-profile-nav [href$="?tab=projects"]' // In user profiles
+	].join());
+
+	if (!projectsTab || select('.Counter', projectsTab)!.textContent!.trim() === '0' || !await hasProjects()) {
```

Comment:
Don't copy-paste without reading it. This code does not run "in user profiles"

Also, it should be the opposite: the tab can have a 0 but there might still be resolved projects.

I think:

```suggestion
	const hasActiveProjects = parseInt(select('[data-hotkey="g b"] .Counter')?.textContent, 10) > 0;
	if (!hasActiveProjects && !await hasAnyProjects()) {
```